src/includes/beta-note-session-replay.mdx

```diff
@@ -1,5 +1,16 @@
 <Note>
 
-Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
+Session Replay is currently in beta. Beta features are still in-progress and may have bugs. We recognize the irony. If you have any questions or feedback, please open a [GitHub issue](https://github.com/getsentry/sentry/issues/new/choose). Alternatively, you can email us at [session-replay@sentry.io](mailto:session-replay@sentry.io).
```

Comment:
Should we ask them to also include a link to the replay and/or the page where they are having issues?

Reply:
that's a good idea! also realized this beta note is on SDK docs, so going to change the repo link to `sentry-javascript`